fio_wrapper/trigger_fio.py

```diff
@@ -57,10 +57,28 @@ def _document_payload(self, data, user, uuid, sample, list_hosts, end_time, fio_
                 document['global_options'] = fio_jobs_dict['global']
             processed.append(document)
             if result['jobname'] != 'All clients':
-                start_time= (int(end_time) * 1000)
-                fio_starttime[result['hostname']] = start_time
+                
+                ramp_time = 0 
+                if 'ramp_time' in result['job options']:
+                    ramp_time = int(result['job options']['ramp_time'])
+                elif 'ramp_time' in document['global_options']:
```

Comment:
just a suggestion: this should be an if, or have an else clause as well for code readability, not blocking the pr for this.